funcr/funcr_test.go

```diff
@@ -95,6 +95,11 @@ func TestPretty(t *testing.T) {
 			{"nine", "three"},
 			{"seven", "six"},
 		},
+		struct {
+			X *int
+		}{
+			X: (*int)(nil), // typed nil
+		},
```

Comment:
It would be useful to have more test cases:
- non-nil pointer (to ensure that it works)
- nil interface
- non-nil interface

Reply:
added 👍

Reply:
Thanks. You can squash directly, for such a simple change it's easy to remember what changed and for more complex ones the Github diff for force-pushes works well.

Reply:
did it!